schema-mapping/data-types/core/Code.json

```diff
@@ -0,0 +1,18 @@
+{
+  "identifier": "core:Code",
+  "table": "property",
+  "value": {
+    "column": "val_string",
+    "type": "string"
+  },
+  "properties": [
+    {
+      "name": "codeSpace",
+      "namespace": "http://3dcitydb.org/3dcitydb/core/5.0",
+      "value": {
+        "column": "val_codespace",
+        "type": "string"
```

Comment:
here the type should be `uri`?

Reply:
Yes. Fixed with f0919f27cf05be5859ab60e9e9aff3903f753c2c